src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -680,5 +705,15 @@ private static AzureEditionDetails[] GetAzureMaxSizes(IEnumerable<AzureEdition>
             }
             return sizes.ToArray();
         }
+
+        /// <summary>
+        /// Converts value in KBs to MBs with two decimal places
+        /// </summary>
+        /// <param name="valueInKb">value in kilo bytes</param>
+        /// <returns>Returns as String</returns>
+        private string ConvertKbtoMbString(double valueInKb)
+        {
+            return (Math.Round(valueInKb / 1000, 2)).ToString("0.00") + " MB";
```

Comment:
1. This should just be a static function - preferably in a general utils class
2. Same as above - we shouldn't pass the units and instead add those in on the UI side

Reply:
Also looks like we divide database sizes by 1024 in other code, like DatabasePrototype or DbSize, when converting to larger units